src/components/Table/TableViewDropdown/__snapshots__/TableViewDropdown.story.storyshot

```diff
@@ -136,409 +108,17 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Table
                 </title>
               </svg>
             </div>
-          </div>
+          </button>
           <div
-            aria-labelledby="iot--view-dropdown"
+            aria-labelledby="downshift-13-label"
             className="bx--list-box__menu"
-            id="iot--view-dropdown__menu"
+            id="downshift-13-menu"
+            onBlur={[Function]}
+            onKeyDown={[Function]}
+            onMouseLeave={[Function]}
             role="listbox"
-          >
```

Comment:
Do you know why this massive chunk got deleted?

Reply:
Yep, the list box is no longer rendered to the DOM until the list box has been actually opened. From my understanding it's an a11y improvement that is tied into the new version of Downshift that Carbon upgraded to.